src/main/kotlin/Project2/Hello.kt

```diff
@@ -0,0 +1,5 @@
+package project2

```

Comment:
Du hast den Ordnernamen nicht klein geschrieben. Tipp für Windows: Erst in was anderes umbenennen (z.B. `Project2` ~> `Project2_`) und dann in die gewünschte Schreibweise (`Project2_` ~> `project2`). So umgeht man das Problem, dass Windows `Project2` und `project2` für das gleiche hält.